test/models/user_test.rb

```diff
@@ -0,0 +1,10 @@
+require 'test_helper'
+
+class UserTest < ActiveSupport::TestCase
+  test "approve a user" do
+    user = User.create!(name: 'New Lawyer', email: 'new@layer.com', password: 'password')
```

Comment:
Line is too long. [89/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.